src/NHibernate.TestDatabaseSetup/TestDatabaseSetup.cs

```diff
@@ -102,8 +102,17 @@ private static void SetupSqlServerOdbc(Cfg.Configuration cfg)
 
 		private static void SetupFirebird(Cfg.Configuration cfg)
 		{
+			Directory.CreateDirectory(@"D:\SqlData\Firebird");
```

Comment:
So I may just locate the file on d:\ root if that prove to be any better than putting it within Firebird installation folder. That would avoid that line. Actually I was seeing `TestDatabaseSetup` as a Teamcity exclusive thing, managing myself my own local db. So I would avoid to do such changes now.